mmv1/products/cloudfunctions2/api.yaml

```diff
@@ -274,29 +274,50 @@ objects:
           An Eventarc trigger managed by Google Cloud Functions that fires events in
           response to a condition in another service.
         properties:
-          - !ruby/object:Api::Type::String 
+          - !ruby/object:Api::Type::String
             name: 'trigger'
             description: 'The resource name of the Eventarc trigger.'
             output: true
-          - !ruby/object:Api::Type::String 
+          - !ruby/object:Api::Type::String
             name: 'triggerRegion'
             description: |
               The region that the trigger will be in. The trigger will only receive
               events originating in this region. It can be the same
               region as the function, a different region or multi-region, or the global
               region. If not provided, defaults to the same region as the function.
-          - !ruby/object:Api::Type::String 
+          - !ruby/object:Api::Type::String
             name: 'eventType'
             description: 'Required. The type of event to observe.'
-          - !ruby/object:Api::Type::String 
+          - !ruby/object:Api::Type::Array
+            name: 'eventFilters'
+            description: 'Criteria used to filter events.'
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: attribute
+                  description: |
+                    'Required. The name of a CloudEvents attribute.
+                    Currently, only a subset of attributes are supported for filtering. Use the `gcloud eventarc providers describe` command to learn more about events and their attributes.
+                    Do not filter for the 'type' attribute here, as this is already achieved by the resource's `event_type` attribute.
+                - !ruby/object:Api::Type::String
+                  name: value
```

Comment:
I manually tested changing the `value` (as there are not other possible values of `attribute` in the context of a GCS event) and it can change ok 👍 

I could imagine an argument that a cloud function being triggered in a different way is a fundamentally 'different' cloud function than the old version, so a ForceNew and replacing the whole set of resources that make up the function + its trigger could makes sense. My gut feeling is to not do `input:true` because the 1 resource that needs changing can be updated, but I'm not strongly holding onto that belief